_rules/heading-descriptive-b49b2e.md

```diff
@@ -55,8 +50,11 @@ Implementation of [Presentational Roles Conflict Resolution][] varies from one b
 
 ## Background
 
+Headings that are in the accessibility tree and not visible are a failure of [Success Criterion 1.3.1 Info and Relationships][sc131]. These are not tested by this rule but they can still fail [Success Criterion 2.4.6 Headings and Labels][sc246]. 
```

Comment:
I don't think the first sentence is true. 1.3.1 state:
> Information, [structure](https://www.w3.org/TR/WCAG21/#dfn-structure), and [relationships](https://www.w3.org/TR/WCAG21/#dfn-relationships) conveyed through [presentation](https://www.w3.org/TR/WCAG21/#dfn-presentation) can be [programmatically determined](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable) or are available in text.

Headings that are not visible do not convey anything through presentation and therefore cannot fail 1.3.1 🤔

Reply:
Many times headings are visually implicit thanks to the design and layout of the page (and might be visually unpleasant), but might be very helpful for AT users. SR-only headings are not necessary failures of WCAG.

Reply:
@WilcoFiers and I worked on this today and changes have been made.